pkg/repository/v1manifest/local_manifests.go

```diff
@@ -91,13 +91,20 @@ func (ms *FsManifests) LoadManifest(role ValidManifest) (bool, error) {
 }
 
 // LoadComponentManifest implements LocalManifests.
-func (ms *FsManifests) LoadComponentManifest(filename string) (*Component, error) {
-	var role Component
-	exists, err := ms.load(filename, &role)
-	if !exists {
-		return nil, nil
+func (ms *FsManifests) LoadComponentManifest(index *Index, filename string) (*Component, error) {
+	com := new(Component)
```

Comment:
Could you continue to use `load` and move the call to `ReadManifest` into `LoadManifest`?